src/ctim/schemas/weakness.cljc

```diff
@@ -107,14 +109,14 @@
     (f/entry :method v/DetectionMethod
              :description (str "identifies the particular detection method "
                                "being described"))
-    (f/entry :description c/MedString
+    (f/entry :description c/Markdown
              :description (str "provide some context of how this method can "
                                "be applied to a specific weakness")))
    (f/optional-entries
     (f/entry :effectiveness v/DetectionEffectiveness
              :description (str "how effective the detection method may be in "
                                "detecting the associated weakness"))
-    (f/entry :effectiveness_notes c/MedString
+    (f/entry :effectiveness_notes c/Markdown
```

Comment:
```
(->> "/var/cwe/1000.xml"
       input-stream
       xml-input-stream->weakness-records
       (map :detection-methods)
       (mapcat identity)
       (map :detection-method)
       (map :effectiveness-notes)
       (remove nil?)
       (map type)
       set) => #{java.lang.String}
```
This change to `:effectiveness_notes` is not needed, there are no cases in the production data where it uses anything other than a simple string.